### PR TITLE
Feature/create marathon metric

### PR DESCRIPTION
**Pull request by mflilian**

### CODE REVIEW

File: model/interfaces.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+type MetricsReporter interface {
+	StartSegment(string) map[string]interface{}
+	EndSegment(map[string]interface{}, string)
+
+	StartDatastoreSegment(datastore, collection, operation string) map[string]interface{}
+	EndDatastoreSegment(map[string]interface{})
+
+	StartExternalSegment(string) map[string]interface{}
+	EndExternalSegment(map[string]interface{})
+}
```
Comment (bitbanshee) [inline model/interfaces.go:12]: Is this file needed?

File: model/mixed_metrics_reporter.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		return f()
+	}
+
+	for _, mr := range m.MetricsReporters {
+		data := mr.StartSegment(name)
+		defer mr.EndSegment(data, name)
+	}
+
+	return f()
+}
```
Comment (bitbanshee) [inline model/mixed_metrics_reporter.go:28]: Is this file needed?

File: Makefile [hunk 3]
```diff
@@ -48,7 +47,7 @@ cross: assets
 	@echo "Building for linux-i386..."
 	@env GOOS=linux GOARCH=386 go build -o ${BIN_PATH}/${BIN_NAME}-linux-i386
 	@echo "Building for linux-x86_64..."
-	@env GOOS=linux GOARCH=amd64 go build -o ${BIN_PATH}/${BIN_NAME}-linux-x86_64
+	@env GOOS=linux GOARCH=amd64 go buld -o ${BIN_PATH}/${BIN_NAME}-linux-x86_64
```
Comment (thiagomachado) [inline Makefile:50]: This should be `build`, right?
